tests/UnitTests/Expect.fs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Pulsar.Client.UnitTests
+
+open System
+open Expecto
+open Expecto.Flip
+
+module internal Expect =
+
+    let throwsTh<'texn> f =
```

Comment:
Can you please rename it or make private? I can't guess what throwsTh means =(

Reply:
Ok. Now it named 'throwsT2'. Because original Expecto function has name 'throwsT' and 'throwsT2' extends 'throwsT' behavior ,'throwsT2' follows same naming pattern.